test/unit/server/handlers/thirdpartyRequestsVerifications.test.ts

```diff
@@ -49,13 +49,13 @@ const PostThirdpartyRequestsVerificationsRequest = {
   },
   params: {},
   payload: {
-    //TODO:
+    // TODO:
```

Comment:
TODO what? Either we need a follow-up story to do whats needed here or remove this

Reply:
removed.